sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -908,6 +915,182 @@ class RotatingCertificateCredential implements TokenCredential {
+
+```
+import { useIdentityExtension, InteractiveBrowserCredential, InteractiveBrowserCredentialOptions } from "@azure/identity";
+import { cachePersistenceExtension } from "@azure/identity-cache-persistence";
+
+useIdentityExtension(cachePersistenceExtension);
+
+const AUTH_RECORD_PATH = "./tokencache.bin";
+const readFileAsync = promisify(fs.readFile);
+const fileContent = await readFileAsync(path.join(process.cwd(), AUTH_RECORD_PATH),{ encoding: "utf-8" });
+const authRecord: AuthenticationRecord= deserializeAuthenticationRecord(fileContent);
+
+const options: InteractiveBrowserCredentialOptions = {
+  tokenCachePersistenceOptions: {
+    enabled: true
+  },
+  authenticationRecord: authRecord
+};
+const credential = new InteractiveBrowserCredential(options);
+
+```
+
+The credential created in this example will silently authenticate given that a valid token for corresponding to the `AuthenticationRecord` still exists in the persisted token data. There are some cases where interaction will still be required such as on token expiry, or when additional authentication is required for a particular resource.
+
+### Persisting credentials by configuring TokenCachePersistenceOptions
+
+Many credential implementations in the `@azure/identity` library have an underlying token cache which persists sensitive authentication data such as account information, access tokens, and refresh tokens. By default this data exists in an in memory cache which is specific to the credential instance. However, there are scenarios where an application needs persist it across executions in order to share the token cache across credentials. To accomplish this the `@azure/identity` provides the `tokenCachePersistenceOptions`.
+
+> IMPORTANT! The token cache contains sensitive data and MUST be protected to prevent compromising accounts. All application decisions regarding the persistence of the token cache must consider that a breach of its content will fully compromise all the accounts it contains.
+
+#### Using the default token cache
+
+The simplest way to persist the token data for a credential is to to use the default `tokenCachePersistenceOptions`. This will persist and read token data from a shared persisted token cache protected to the current account.
+
+```
+import { useIdentityExtension, InteractiveBrowserCredential, InteractiveBrowserCredentialOptions } from "@azure/identity";
+import { cachePersistenceExtension } from "@azure/identity-cache-persistence";
+
+useIdentityExtension(cachePersistenceExtension);
+
+  const options: InteractiveBrowserCredentialOptions =  {
+    tokenCachePersistenceOptions: {
+      enabled: true
+    }
+  }
+  const credential = new InteractiveBrowserCredential(options);
```

Comment:
```suggestion
import { useIdentityExtension, InteractiveBrowserCredential } from "@azure/identity";
import { cachePersistenceExtension } from "@azure/identity-cache-persistence";

useIdentityExtension(cachePersistenceExtension);

const credential = new InteractiveBrowserCredential({
  tokenCachePersistenceOptions: {
    enabled: true
  }
});
```

This seems simple to me. And there were some extra spaces at the beginning of some lines!

Reply:
Do you mind applying this change to other instances in this pull request where the options might be separated from the function that they’re being sent to? Unless separating them has some benefit I might not be seeing! Let me know.

Reply:
I thought if they use options they will get intellisense